etl/churn.kp/knowledge.md

```diff
@@ -501,15 +523,19 @@ def compute_churn_week(df, week_start, enable_upload_csv=False):
             .sum('subsession_length')
             .withColumnRenamed('sum(subsession_length)', 'usage_seconds')
     )
-
+    
     # Append this column to the original data frame.
```

Comment:
nit: since you've moved the |get_newest_per_client|, I think you should update the comment too